src/Uno.UI/UI/Xaml/Media/CompositionTarget.iOS.cs

```diff
@@ -40,7 +41,7 @@ public  partial class CompositionTarget
 
 		private static void OnFrame()
 		{
-			_handlers.ForEach(h => h(null, null));
+			_handlers.ToList().ForEach(h => h(null, null));
```

Comment:
Why do you force a materialization of the list here?

Reply:
There's a race condition with the handlers list. It's not used much there yet.